src/bindings/python/src/pyopenvino/core/offline_transformations.cpp

```diff
@@ -129,63 +117,4 @@ void regmodule_offline_transformations(py::module m) {
             manager.run_passes(model);
         },
         py::arg("model"));
-
-    // todo: remove as serialize as part of passManager api will be merged
-    m_offline_transformations.def(
```

Comment:
Shouldn't it start with deprecation notice and removing it later? Won't it break compatibility?

Reply:
offline transformations are considered as internal module which shouldn't be used by the end user.